src/GovUk.Education.ExploreEducationStatistics.Content.Model/Contact.cs

```diff
@@ -1,23 +1,18 @@
+#nullable enable
 using System;
 using System.ComponentModel.DataAnnotations;
 
-namespace GovUk.Education.ExploreEducationStatistics.Content.Model
+namespace GovUk.Education.ExploreEducationStatistics.Content.Model;
+
+public class Contact
 {
-    public class Contact
-    {
-        [Key] [Required] public Guid Id { get; set; }
+    [Key] [Required] public Guid Id { get; set; }
 
-        [Required]
-        public string TeamName { get; set; }
+    [Required] public string TeamName { get; set; } = null!;
 
-        [Required]
-        [EmailAddress]
-        public string TeamEmail { get; set; }
+    [Required] [EmailAddress] public string TeamEmail { get; set; } = null!;
 
-        [Required]
-        public string ContactName { get; set; }
+    [Required] public string ContactName { get; set; } = null!;
 
-        [Required]
-        public string ContactTelNo { get; set; }
-    }
-}
\ No newline at end of file
+    public string? ContactTelNo { get; set; }
```

Comment:
My first thought is that none of these DataAnnotations should be required, since EF will recognise the `Id` field is the key and nullability of the fields is handled automatically too now that you've added the nullable context to this class.

I see that there's one request `PublicationController.UpdateContact` using this type in the request body.

I think I would look at making that use the `ContactSaveViewModel` and then strip all the annotations from this class.

Reply:
I've removed the attributes from `Contact`, but I've switched to use `ContactViewModel`. I don't think we need the attributes on the class when returning the view model. And `ContactSaveViewModel` seems to represent a request, so I've also renamed and moved `ContactSaveViewModel` to be `ContactSaveRequest`.

Reply:
I also created a dummy migration to make sure this hasn't changed anything, and it was blank, so all looks good.